### PR TITLE
test for division

### PR DESCRIPTION
**Pull request by debbycosoi**

### CODE REVIEW

File: tests/test_for_division.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+class CalculatorTests(unittest.TestCase):
+
+	@classmethod
+	def setUpClass(cls):
+		"""
+		instantiate our class  (fixture)
+		"""
+		cls.calc = Calculator()
+
```
Comment (dd-code-immersives) [inline tests/test_for_division.py:12]: add one more test to handle zero case

**Review by dd-code-immersives — CHANGES_REQUESTED** [review]
